client/src/Components/Deepdive/Deepdive.js

```diff
@@ -1,19 +1,50 @@
 import React from "react";
 import * as S from "./Deepdive.style";
-import { ReactComponent as ParkDetail } from "../../content/park-foreground.svg";
 import { Button } from "../Buttons/Button.js";
 const Deepdive = ({ data }) => {
- 
+  const [scrollPosition, setScrollPosition] = React.useState(null);
+  const [foregroundOpacity, setForegroundOpacity] = React.useState(1);
+
+  window.onscroll = () => {
+    setScrollPosition(window.scrollY);
+    console.log("The scroll position: ", scrollPosition);
```

Comment:
Hate to be dat guy: we should start culling these bad boys (console.logs) henceforth